tools/internal_ci/linux/grpc_aws_experiment.sh

```diff
@@ -29,6 +45,12 @@ sudo apt update && sudo apt install -y jq
 AWS_MACHINE_IMAGE=ami-026141f3d5c6d2d0c
 AWS_INSTANCE_TYPE=t4g.xlarge
 AWS_SECURITY_GROUP=sg-021240e886feba750
+# Max allowed lifespan of the AWS instance. After this period of time, the instance will
+# self-terminate (delete itself).
+AWS_INSTANCE_MAX_LIFESPAN_MINS=120
+# increase the size of the root volume so that builds don't run out of disk space
+AWS_STORAGE_SIZE_GB=75
+AWS_DEVICE_MAPPING="DeviceName='/dev/sda1',Ebs={VolumeSize=${AWS_STORAGE_SIZE_GB}}"
```

Comment:
Did this work for you? When I tried sda1 the device wasnt actually created. /dev/sda1 is the normal root of the system.

Reply:
yes it does. It actually changes the default size of the root filesystem (normally 10GB, determined by the image) to whatever you specify here. I ran `df -h` on the workers and that confirmed it. Also stuff that was failing before is now passing.